### ISSUE TITLE
Cannot update beyond 2020.7 (3)

### ISSUE THREAD

**Issue by PierreScerri** [home-assistant/iOS#1215]
**Device model, version and app version**

Model Name: iMac 24" Early 2009
macOS Version: Catalina 10.15.3
App Version: 2020.7 (3)
Safari Version:  13.0.5 (15608.5.11)

**Home Assistant Core Version**

0.116.4
**Describe the bug**
When I update to a Companion App version higher than 2020.7 (3) the app will start loading but quits after a few seconds.

**To Reproduce**
Update to Companion app version higher than 2020.7 (3)

**Expected behavior**
Companion app should load.

**Screenshots**

**Additional context**

**Comment by zacwest** [comment]
Can you please attach a crash log from your device? This can be found in ~/Library/Logs/DiagnosticReports or ~/Library/Logs/CrashReporter.

**Comment by PierreScerri** [comment]
There is nothing there.

**Comment by PierreScerri** [comment]
Still not working with 2020.7 (10)

**Comment by PierreScerri** [comment]
I have found that when I update and run the latest version there is still the previous version running.
I uninstalled the app and re installed the latest update and it is working even if it is hogging the CPU.

**Comment by zacwest** [comment]
Ah, that makes sense! This is actually a weird part about how Catalyst apps run, and sometimes occurs. I haven't seen it on Big Sur at all, so at least it's probably easier to understand going forward.

If you're seeing it hog the CPU and it's the Home Assistant.app process, please open another issue and attach a Sample via:

1. Launch Activity Monitor from /Applications/Utilities
1. Find Home Assistant.app in the list and double-click it
1. Click the "Sample" button at the bottom
1. Let it run and then attach the Sample on GitHub

The Sample doesn't contain any personal information; it's just what the app was doing at any particular time.

**Comment by PierreScerri** [comment]
Perhaps I did not explain myself well.
The home assistant app itself does not hog the CPU, however, a com.apple.WebKit.WebContent instance is launched which then hogs the CPU.
When I quit the home assistant app, the com.apple.WebKit.WebContent process also terminates and the CPU is released.

**Comment by zacwest** [comment]
High usage in the webview process points to an issue in the front end. I believe there may be some issues with the camera cards. Worth checking in with the [front end repo/devs](https://www.home-assistant.io/help/).

**Comment by PierreScerri** [comment]
Thank you for the link. I’ll have a look. However, I’m not using any camera cards. Are there any other cards that may cause the same issue?

**Comment by PierreScerri** [comment]
I did some troubleshooting on the use of CPU.

I have identified this card https://github.com/bramkragten/weather-card as the main cause of the increase in CPU usage.

I tried it on Safari and Brave(Chrome) browsers and the CPU increases to 60-80% when viewing a tab that has this card, whereas using the app increases it to 130-150%.

Other cards that have some effect are https://github.com/mishaaq/sun-card and using iframe to show a windy.com weather chart.

Moving to tabs that do not have these cards releases the CPU.
